src/components/view/ListView.vue

```diff
@@ -16,220 +16,272 @@
+          @click="saveValue(record)"
+          v-if="editableValueKey === record.key" >
+          <a-icon type="check-circle" theme="twoTone" twoToneColor="#52c41a" />
         </a-button>
-      </a-tooltip>
-      <a-tooltip placement="top">
-        <template slot="title">Move up one row</template>
-        <a-button shape="round" @click="moveItemUp(record)" class="shift-btn">
-          <a-icon type="caret-up" class="shift-btn" />
-        </a-button>
-      </a-tooltip>
-      <a-tooltip placement="top">
-        <template slot="title">Move down one row</template>
-        <a-button shape="round" @click="moveItemDown(record)" class="shift-btn">
-          <a-icon type="caret-down" class="shift-btn" />
+        <a-button
+          shape="circle"
+          size="default"
+          @click="editableValueKey = null"
+          v-if="editableValueKey === record.key" >
+          <a-icon type="close-circle" theme="twoTone" twoToneColor="#f5222d" />
         </a-button>
-      </a-tooltip>
-    </div>
-
-    <template slot="value" slot-scope="text, record">
-      <a-input
-        v-if="editableValueKey === record.key"
-        :defaultValue="record.value"
-        :disabled="!('updateConfiguration' in $store.getters.apis)"
-        v-model="editableValue"
-        @keydown.esc="editableValueKey = null"
-        @pressEnter="saveValue(record)">
-      </a-input>
-      <div v-else style="width: 200px; word-break: break-all">
-        {{ text }}
-      </div>
-    </template>
-    <template slot="actions" slot-scope="text, record">
-      <a-button
-        shape="circle"
-        :disabled="!('updateConfiguration' in $store.getters.apis)"
-        v-if="editableValueKey !== record.key"
-        icon="edit"
-        @click="editValue(record)" />
-      <a-button
-        shape="circle"
-        :disabled="!('updateConfiguration' in $store.getters.apis)"
-        @click="saveValue(record)"
-        v-if="editableValueKey === record.key" >
-        <a-icon type="check-circle" theme="twoTone" twoToneColor="#52c41a" />
-      </a-button>
-      <a-button
-        shape="circle"
-        size="default"
-        @click="editableValueKey = null"
-        v-if="editableValueKey === record.key" >
-        <a-icon type="close-circle" theme="twoTone" twoToneColor="#f5222d" />
-      </a-button>
-    </template>
-  </a-table>
+      </template>
+      <template slot="tariffActions" slot-scope="text, record">
+        <a-button
+          shape="circle"
+          v-if="editableValueKey !== record.key"
+          icon="edit"
+          @click="editTariffValue(record)" />
+      </template>
+    </a-table>
+    <a-modal
+      :visible="showTariffAction"
+      :closable="true"
+      :title="$t('label.quota.configuration')"
+      :confirmLoading="actionLoading"
+      style="top: 20px;"
+      @ok="submitTariff"
+      @cancel="() => { this.showTariffAction = false }"
+      centered
+    >
+      <a-form
```

Comment:
@utchoang can you undo changes in this component; instead any additional things could to pushed into it using slots (see other examples using `<slot ...`